documentation/release_notes.rst

```diff
@@ -8,6 +8,61 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.5
+=============
+
+New Features
+------------
+- Added new random number distributions: ``exponential_distribution``, ``bernoulli_distribution``,
+  ``geometric_distribution``, ``lognormal_distribution``, ``weibull_distribution``, ``cachy_distribution``, ``extreme_value_distribution``.
+- Added the serial-based versions of the following algorithms: ``all_of``, ``any_of``, 
+  ``none_of``, ``count``, ``count_if``, ``for_each``, ``find``, ``find_if``, ``find_if_not``.
+  For the detailed list, please refer to `Tested Standard C++ API Reference`_. 
+- Improved performance of ``search`` and ``find_end`` algorithms on GPU devices.
+
+Fixed Issues
+------------
+- Fixed SYCL* 2020 features deprecation warnings.
+- Fixed some corner cases of ``normal_distribution`` functionality.
+- Fixed a floating point exception occurring on CPU devices when a program uses a lot of oneDPL algorithms and DPC++ kernels.
+- Fixed hanging of reduction-based algorithms (``count``, ``count_if``, ``is_partitioned``, ``lexicographical_compare``, ``max_element``, ``min_element``, ``minmax_element``,    ``reduce``, ``transform_reduce``) used with DPC++ policies. The hanging occurred when the algorithms handled elements with a large size (e.g. 64 bytes and more) and a program used a lot of oneDPL algorithms and DPC++ kernels.
+- Fixed a possible data race for the following algorithms with DPC++ policies: ``count``, ``count_if``, ``is_partitioned``,
+  ``lexicographical_compare``, ``max_element``, ``min_element``, ``minmax_element``, ``transform_reduce``. 
```

Comment:
Probably we can unite these bullets omitting extra information in the first one.

- Fixed possible hanging and data races of the following algorithms used with DPC++ execution policies: ``count``, ``count_if``, ``is_partitioned``, ``lexicographical_compare``, ``max_element``, ``min_element``, ``minmax_element``, ``transform_reduce``.

Reply:
> Probably we can unite these bullets omitting extra information in the first one.
> 
> * Fixed possible hanging and data races of the following algorithms used with DPC++ execution policies: `count`, `count_if`, `is_partitioned`, `lexicographical_compare`, `max_element`, `min_element`, `minmax_element`, `transform_reduce`.

Merged the two bullets.